Resources/Prototypes/Entities/Objects/Misc/paper.yml

```diff
@@ -213,3 +210,188 @@
     - state: folder-colormap
       color: "#3f3f3f"
     - state: folder-base
+
+- type: entity
+  name: mime's rubber stamp
+  parent: BaseItem
+  id: RubberStampMime
+  description: A rubber stamp for stamping important documents.
+  components:
+  - type: Stamp
+    stampedName: "Mime"
```

Comment:
as said before these need to take in loc strings and then resolve them rather than just taking in strings, otherwise ruskis will get mad

Reply:
Huh, didn't actually occur to me for the names, will seperate them out.